lib/chewy/search/response.rb

```diff
@@ -19,6 +19,12 @@ def hits
         @hits ||= hits_root['hits'] || []
       end
 
+      # Raw response body obtained from ES.
+      # @return [Hash, nil]
+      def body
```

Comment:
Is this method needed?

Reply:
Add it to have the visibility of the actual response from ES in leadgein repo